server/plugins/authentication.js

```diff
@@ -0,0 +1,148 @@
+const { makeExtendSchemaPlugin, gql } = require("graphile-utils");
+
+const PassportLoginPlugin = makeExtendSchemaPlugin(build => ({
+  typeDefs: gql`
+    input RegisterInput {
+      username: String!
+      email: String!
+      password: String!
+      name: String
+      avatarUrl: String
+    }
+
+    type RegisterPayload {
+      user: User! @pgField
+    }
+
+    input LoginInput {
+      username: String!
+      password: String!
+    }
+
+    type LoginPayload {
+      user: User! @pgField
+    }
+
+    extend type Mutation {
+      register(input: RegisterInput!): RegisterPayload
+      login(input: LoginInput!): LoginPayload
+    }
+  `,
+  resolvers: {
+    Mutation: {
+      async register(
+        mutation,
+        args,
+        context,
+        resolveInfo,
+        { selectGraphQLResultFromTable }
+      ) {
+        const { username, password, email, name, avatarUrl } = args.input;
+        const { rootPgPool, login, pgClient } = context;
+        try {
+          // Call our login function to find out if the username/password combination exists
```

Comment:
Looks like this comment was copy-pasted from the `login` function. Can you update it to be accurate for the `register` function instead?